airbyte-webapp/.eslintrc

```diff
@@ -41,15 +57,27 @@
         }
       }
     ],
+    "@typescript-eslint/array-type": "warn",
```

Comment:
This is more a personal taste. I feel `Array<{ complex: 'type'; foo: number }>` a bit easier to read for more complex types than the `[]` equivalent. Though I don't have strong feelings around this, so happy to go with either way.

Reply:
This is also a personal thing, but I'm also open to either force one direction or hybrid.

One I was thinking about is `array-simple` instead. Only simple types should use `[]` instead of `Array`. Thoughts?
https://typescript-eslint.io/rules/array-type/

Reply:
`array-simple` sounds like my preferred way. But not extremly strong feelings :)